CS483/CS483/Kartaclysm/Services/IO/PlayerInputMapping.cpp

```diff
@@ -359,7 +359,7 @@ namespace Kartaclysm
 	//--------------------------------------------------------------------------------
 	int PlayerInputMapping::GetFirstAvailableInput()
 	{
-		for (int i = 0; i <= GLFW_JOYSTICK_LAST + 4; i++)
+		for (int i = 0; i <= GLFW_JOYSTICK_LAST + 8; i++)
 		{
 			if (PlayerUsingInput(i) == -1)
```

Comment:
I think I was playing with numbers trying to fix things in the controller code. I think this might be a leftover thing that didn't actually solve anything.